.github/actions/nodejs/action.yaml

```diff
@@ -17,33 +17,16 @@ runs:
       shell: bash
       run: npm config set audit false
 
-    - name: Configure AWS credentials and bucket region
-      uses: aws-actions/configure-aws-credentials@v4
-      if: contains(runner.name, 'ubuntu-platform')
-      with:
-        aws-region: eu-west-1
-
     - name: Cache NPM build artifacts (S3 bucket cache)
-      uses: everpcpc/actions-cache@v1
-      if: contains(runner.name, 'ubuntu-platform')
+      uses: strophy/actions-cache@opendal-update
```

Comment:
We can't until this issue is fixed and everpcpc is updated https://github.com/apache/incubator-opendal/issues/3562

Reply:
Upstream already has restore/save actions available, and I'm also using OpenDAL in my fork. Why does it work?

Reply:
It has js scripts but not actions. Please compare https://github.com/strophy/actions-cache/tree/opendal-update
and https://github.com/everpcpc/actions-cache 

Our fork works because I've compiled a module for arm64 linux gnu manually.

Reply:
Got it, resolved